src/components/Toolbelt/Toolbelt.js

```diff
@@ -22,19 +24,22 @@ export const Toolbelt = ({
           alt: 'A watering can for hydrating plants.',
           fieldMode: WATER,
           toolImageId: 'watering-can',
+          hiddenText: 'select watering can',
```

Comment:
Thank you for prioritizing accessibility! Farmhand certainly has a lot of ground to cover in this regard, and this is a great way to get us on the right track.

Out of curiosity, are these `hiddenText` strings all lowercased intentionally? It's fine either way since they will not be visually read, I just want to know if this was done as a matter of best practice or otherwise.

Reply:
glad you brought this up actually, i just kinda threw them down here thinking about the bigger picture of this PR but we should think about this before merging. I don't actually know if casing matters (other than you definitely don't want all uppercase), so we probably want proper grammar here. this is what a screen reader would say out loud to tell the person what action will be taken, maybe we should do something more like "Select the watering can to water crops", and similarly for the others. that would be similar to what the tooltip does for sighted users. what do you think @jeremyckahn ?

Reply:
i went ahead and updated so you can just comment on them instead, hope that's ok!

Reply:
Looks great, thanks for making the changes! 😃